CHANGELOG.md

```diff
@@ -3,13 +3,143 @@
+resulting in some introspection queries not working.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/1240
+
+### Update the scaffold template so that it works with streams ([#1247](https://github.com/apollographql/router/issues/1247))
+Release v0.9.4 changed the way we deal with Response objects, which can now be streams.
+The scaffold template has been updated so that it generates plugins that are compatible with the new Plugin API.
+
+By [@o0Ignition0o](https://github.com/o0Ignition0o) in https://github.com/apollographql/router/pull/1248
+
+### Create the ExecutionResponse after the primary response was generated ([PR #1260](https://github.com/apollographql/router/pull/1260))
+The `@defer` preliminary work has a surprising side effect: when using methods like `RouterResponse::map_response`, they are
+executed before the subgraph responses are received, because they work on the stream of responses.
+This PR goes back to the previous behaviour by awaiting the primary response before creating the ExecutionResponse.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/1260
+
+### Use the API schema to generate selections ([PR #1255](https://github.com/apollographql/router/pull/1255))
+When parsing the schema to generate selections for response formatting, we should use the API schema instead of the supergraph schema.
+
+By [@Geal](https://github.com/Geal) in https://github.com/apollographql/router/pull/1255
+
+## 📚 Documentation
+
+### Update README link to the configuration file  ([PR #1208](https://github.com/apollographql/router/pull/1208))
+As the structure of the documentation has changed, the link should point to the `YAML config file` section of the overview.
+
+By [@gscheibel](https://github.com/gscheibel in https://github.com/apollographql/router/pull/1208
+
+
+
 # [0.9.4] - 2022-06-14
 
 ## ❗ BREAKING ❗
 
 
 ### Groundwork for `@defer` support ([PR #1175](https://github.com/apollographql/router/pull/1175)[PR #1206](https://github.com/apollographql/router/pull/1206))
-To prepare for the implementation of the `@defer` directive, the `ExecutionResponse`  and `RouterResponse` types now carry a stream of responses instead of a unique response. For now that stream contains only one item, so there is no change in behaviour. However, the Plugin trait changed to accomodate this, so a couple of steps are required to migrate your plugin so it is compatible with router v0.9.4:
+To prepare for the implementation of the `@defer` directive, the `ExecutionResponse`  and `RouterResponse` types now carry a stream of responses instead of a unique response. For now that stream contains only one item, so there is no change in behaviour. However, the Plugin trait changed to accomodate this, so a couple of steps are required to migrate your plugin so that it is compatible with router v0.9.4:
```

Comment:
Should the end of this paragraph be something like:
"... so that it is compatible with versions of the router >=0.9.4" ?

Reply:
Fixed in [3521214](https://github.com/apollographql/router/pull/1276/commits/352121405002346caaf7944037440daccbd1ce0b)